test/integration/docker-compose.yml

```diff
@@ -6,3 +6,8 @@ services:
     ports:
       - 18334:18334
     command: --miningaddr msRY4KpAJ8o9da1YEASy1j2ACnuzh4SyFs
+  ltcd: 
+    image: michael1011/ltcd
+    ports:
+      - 19334:19334
+    command: --miningaddr msRY4KpAJ8o9da1YEASy1j2ACnuzh4SyFs
```

Comment:
I suppose that this address won't work on the Litecoin regtest network but just leave it for now I can replace it when we actually need a working address